cloudinit/config/cc_ca_certs.py

```diff
@@ -83,16 +88,19 @@ def add_ca_certs(certs):
         util.write_file(CA_CERT_CONFIG, out, omode="wb")
 
 
-def remove_default_ca_certs():
+def remove_default_ca_certs(distro_name=None):
     """
     Removes all default trusted CA certificates from the system. To actually
     apply the change you must also call L{update_ca_certs}.
     """
     util.delete_dir_contents(CA_CERT_PATH)
     util.delete_dir_contents(CA_CERT_SYSTEM_PATH)
     util.write_file(CA_CERT_CONFIG, "", mode=0o644)
-    debconf_sel = "ca-certificates ca-certificates/trust_new_crts select no"
-    subp.subp(('debconf-set-selections', '-'), debconf_sel)
+
+    if distro_name is None or distro_name != 'alpine':
```

Comment:
I originally had the code without the None check but added it because of distro_name defaulting to None due to the testcase not passing a value to this function. See my comment just above relating to changing the function definition.

Reply:
Change made.